packages/react/src/components/dropdown-list/dropdown-list.tsx

```diff
@@ -204,426 +140,258 @@ export const DropdownList: VoidFunctionComponent<DropdownListProps> = ({
-                        data-testid="input"
-                        id={fieldId}
-                        isMobile={isMobile}
-                        disabled={disabled}
-                        name={name}
-                        onBlur={handleBlur}
-                        onChange={handleInputChange}
-                        onFocus={handleFocus}
-                        onKeyDown={handleInputKeyDown}
-                        placeholder={placeholder}
-                        ref={inputRef}
-                        required={required}
-                        searchable={searchable}
-                        type="text"
-                        value={inputValue}
-                        {...dataAttributes /* eslint-disable-line react/jsx-props-no-spreading */}
-                    />
-                    <Arrow
-                        aria-labelledby={fieldId}
-                        type="button"
-                        data-testid="arrow"
-                        tabIndex={-1}
-                        onClick={disabled ? undefined : handleArrowClick}
-                        disabled={disabled}
-                    >
-                        <Icon name={open ? 'chevronUp' : 'chevronDown'} size={device === 'mobile' ? '24' : '16'} />
-                    </Arrow>
-                </InputWrapper>
-                {open && (
-                    <StyledListbox
-                        ariaLabelledBy={fieldId}
-                        ref={listboxRef}
-                        data-testid="listbox"
-                        defaultValue={defaultValue}
-                        focusedValue={focusedValue}
-                        id={`listbox_${fieldId}`}
-                        onChange={handleChange}
-                        onFocusChange={searchable ? undefined : handleFocusedValueChange}
-                        onKeyDown={handleListboxKeyDown}
-                        options={filteredOptions}
-                        value={selectedOptionValue}
-                    />
-                )}
-            </StyledFieldContainer>
-            {skipOption && (
-                <ChooserButton
-                    checked={skipSelected}
-                    data-testid="select-skip-option"
-                    groupName={`${fieldId}_skip`}
-                    onChange={handleSkipChange}
-                    type="radio"
-                    value={skipOption.value}
-                >
-                    {skipOption.label}
-                </ChooserButton>
+                <input type="hidden" name={name} value={selectedOption?.value} data-testid="input" />
+                {selectedOption?.label ?? ''}
+                <Arrow
+                    aria-hidden="true"
+                    data-testid="arrow"
+                    focusable={false}
+                    name={open ? 'chevronUp' : 'chevronDown'}
+                    size={device === 'mobile' ? '24' : '16'}
+                />
+            </Textbox>
+            {open && (
```

Comment:
Je mettrais un peu d'air ici
```suggestion
            </Textbox>

            {open && (
```